integreat_cms/static/src/js/pages/page-clickable-sidebar.ts

```diff
@@ -0,0 +1,23 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const boxes = document.querySelectorAll(".sidebar-box");
+    const cookies = document.cookie.split("; ").map((cookie) => cookie.split("="));
+    for (let i = 0; i < boxes.length; i++) {
+        boxes[i].firstElementChild.addEventListener("click", () => {
+            boxes[i].lastElementChild.classList.toggle("hidden");
+            let cookieValue = "hidden";
+            if (boxes[i].lastElementChild.classList.contains("hidden")) {
+                cookieValue = "hidden";
+            } else {
+                cookieValue = "show";
+            }
+            document.cookie = boxes[i].id + "=" + cookieValue + ";SameSite=Lax";
+        });
+    }
+    for (let j = 0; j < cookies.length; j++) {
+        for (let k = 0; k < boxes.length; k++) {
+            if (cookies[j][0] === boxes[k].id && cookies[j][1] === "hidden") {
+                document.querySelector("#" + boxes[k].id).lastElementChild.classList.add("hidden");
+            }
+        }
+    }
```

Comment:
```suggestion
    cookies.forEach(([id, value]) => {
        document.getElementById(id)?.lastElementChild.classList.add("hidden");
    })
```
I think this loop is a bit inefficient - the cookie already holds the id of the box, so no need to iterate over all boxes just to fine the correct box with the right id.
Also, I think we should not overcomplicate matters and only store the cookie when the box is actually hidden - and just remove the entry when it's shown again.